### PR TITLE
QSA-75 Fixed systems

### PR DESCRIPTION
**Pull request by kcsanad98**
- Added new calculations
- Removed some calculations
- Fixed integration and unit tests accordingly

### CODE REVIEW

File: qsa-calculator-server/src/main/resources/application.yml [hunk 1]
```diff
@@ -53,5 +53,8 @@ logging:
 qsa:
   config:
     uris:
-      - https://qsa.inf.unideb.hu/default/api/config/system
-      - https://qsa.inf.unideb.hu/default/api/config/feature
+      #- https://qsa.inf.unideb.hu/default/api/config/system
+      #- https://qsa.inf.unideb.hu/default/api/config/feature
+      - C:\Users\csanad.kolcsei\Dev\QSA\qsa-feature-config-pack
```
Comment (ZoltanSzilagyiCse) [inline qsa-calculator-server/src/main/resources/application.yml]: Please revert these changes.

**Review by ZoltanSzilagyiCse — CHANGES_REQUESTED** [review]

File: qsa-calculator-server/src/main/resources/application.yml (hunk 1 above)
Comment (kcsanad98) [reply qsa-calculator-server/src/main/resources/application.yml]: Done

Comment (kcsanad98) [reply qsa-calculator-server/src/main/resources/application.yml]: Done

**Review by ZoltanSzilagyiCse — APPROVED** [review]
